cmd/entrypoint/namespaces_test.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// This isn't a great unit test, but it's the best I can think of.
```

Comment:
Maybe document the reasons that it's not great here too in case an enterprising dev comes along with ideas to fix it?  Having said that, if the issues are just that it's linux-only or that it calls out to an external website then I suggest not including this comment.  A linux-only test is fine (imo) for a linux-only feature (until we're proactively supporting bsd / jails?) and we have other tests that expect access to external resources e.g. external git repositories.

Reply:
Good call! Explained the main issue in the comment.